ui/elastic/src/App.js

```diff
@@ -132,11 +132,41 @@ const HitsTable = ({ hits }) => {
 
 const sortVersions = filters => {
   return filters.sort((a, b) => {
-    const majorA = parseInt(a.key.split(".", 1)[0], 10);
-    const majorB = parseInt(b.key.split(".", 1)[0], 10);
-    return majorB - majorA;
-  });
-};
+    const partsA = a.key.split(".")
+    const partsB = b.key.split(".")
+    if (partsA.length < 2 || partsB.length < 2) {
+      // Bogus version, list is last.
+      return 1;
+    }
+    let i = 0;
+    while (partsA[i] === partsB[i]) { // Skip all the parts that are equal.
```

Comment:
Should you check that you don't run off the end of both? (Otherwise, won't you get an infinite loop if you have two elements that are exactly equal?)